ansible/install-apps.yml

```diff
@@ -0,0 +1,34 @@
+---
+
+- name: Install OS-level apps
+  hosts: all
+  become: yes
+  vars:
+  - apt_file: /etc/apt/sources.list.d/google-chrome.list
+  - apt_file_Git: 
+  tasks:
+  - name: Does the Google apt file exist?
+    command: test -f {{apt_file}}
+    register: google_apt_exists
+    ignore_errors: yes
+
+  - name: Add Google Chrome key   
+    shell: echo 'deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main'>/etc/apt/sources.list.d/google-chrome.list
```

Comment:
Following the previous suggestion: use an appropriate module. You should always avoid using `command` and `shell` modules whenever possible!

In this case, there's:
- https://docs.ansible.com/ansible/latest/modules/lineinfile_module.html — it's not very recommended but works for simple things
- https://docs.ansible.com/ansible/latest/modules/copy_module.html — this is better, you can put a text file in the repo and copy it over to the target location
- https://docs.ansible.com/ansible/latest/modules/template_module.html — you can put a jinja2 template into the repo, use a variable there. This way data will be separated.
- https://docs.ansible.com/ansible/latest/modules/apt_repository_module.html — since you're dealing with apt repo and it's a distinct concept, there's this module just for your case. It's the best option here!